src/@utils/schaclSchema.ts

```diff
@@ -0,0 +1,55 @@
+import { retrieveShaclSchema } from '@utils/aquarius'
+import { ShaclSchemaField } from '@context/MarketMetadata/_shaclType'
+
+import { capitalizeFirstLetter } from '@utils/textTransform'
+
+function getMinMax(valueField: string, field: ShaclSchemaField) {
+  let min = field.minLength
+  const max = field.maxLength
+
+  // TODO: remove once all fields in schema has minLength property
+  if (!min) {
+    switch (valueField) {
+      case 'author':
+        min = 1
+        break
+      case 'tags':
+        min = 1
+        break
+    }
+  }
+
+  return {
+    minLength: min,
+    maxLength: max
+  }
+}
+
+export async function validateFieldSchaclSchema(
+  keyField: string,
+  valueField: string,
+  value: any,
+  createError: any
+): Promise<any> {
+  const schemaFields: any = await retrieveShaclSchema()
+  const fieldSchema: ShaclSchemaField = schemaFields[keyField][valueField]
+  const { minLength, maxLength } = getMinMax(valueField, fieldSchema)
+  console.log(schemaFields)
+
+  // TODO: add minLength when integrated in endpoint
```

Comment:
Same applies as above, lets make sure we are tracking when the change has been made